packages/tldraw/src/lib/ui/components/primitives/Button/TldrawUiButton.tsx

```diff
@@ -11,15 +10,6 @@ export interface TLUiButtonProps extends React.HTMLAttributes<HTMLButtonElement>
 /** @public */
 export const TldrawUiButton = React.forwardRef<HTMLButtonElement, TLUiButtonProps>(
 	function TldrawUiButton({ children, disabled, type, ...props }, ref) {
-		const editor = useEditor()
-
-		// If the button is getting disabled while it's focused, move focus to the editor
-		// so that the user can continue using keyboard shortcuts
```

Comment:
As mentioned in the PR desc., it looks like this isn't needed anymore...

Reply:
are you sure? the document.activeElement still reverts to the `body` when the button becomes inactive in my tests. admittedly that no longer seems to prevent keyboard shortcuts from working but maybe it could cause other issues?

Reply:
i think i'm specifically making sure that the original PR (https://github.com/tldraw/tldraw/pull/2630) is working. if it causes other issues, then i think i would rather that we know what those issues are vs. this covering them up potentially (as a fix for something unrelated)